canonical-web-and-design/juju.is
# New homepage what is juju

## Done
Build the "What is Juju" animated section of the homepage

## QA
- Open the demo link
- Scroll down the page and through the "What is Juju" section and see the animated illustrations
- Check different screen sizes.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2017
